app/controllers/import/prospects_controller.rb

```diff
@@ -0,0 +1,15 @@
+module Import
+  class ProspectsController < ApplicationController
+    include ActionView::Helpers::TextHelper
+
+    def create
+      prospects = ProspectApiWrapper.instance.get_prospects
+      created_prospects = Prospect.create(prospects.reverse)
+      size = created_prospects.select(&:id).count
```

Comment:
when `create` receives array then it returns something like `[1, 2, nil, 3, nil]` - `id` for created objects and `nil` for not created. But it is not needed anymore